src/Compilers/VisualBasic/Portable/Scanner/ScannerBuffer.vb

```diff
@@ -95,6 +96,36 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return p
         End Function
 
+        Friend Function TryPeek(at As Integer, <Out()> ByRef ch As Char) As Boolean
```

Comment:
Tried a version with `ChrW(0)` as the default but whilst still buildable it fails the unit tests.